domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/FullAdmissionResponse.java

```diff
@@ -27,13 +28,14 @@
     @Builder
     public FullAdmissionResponse(
         YesNoOption freeMediation,
+        FreeMediation freeMediationWithPhoneNumber,
```

Comment:
doesn't look good, should keep one object FreeMediation with flag and phone or just keep phone number here

Reply:
Please rename param to `freeMediation`

Reply:
doesn't apply after latest change